src/VisualStudio.Roslyn.SDK/ComponentDebugger/Roslyn.ComponentDebugger.csproj

```diff
@@ -30,6 +30,8 @@
     <PackageReference Include="Microsoft.VisualStudio.Composition" Version="$(MicrosoftVisualStudioCompositionVersion)" PrivateAssets="all" />
     <PackageReference Include="Microsoft.VisualStudio.Shell.Interop" Version="$(MicrosoftVisualStudioShellPackagesVersion)" PrivateAssets="all" />
     <PackageReference Include="Microsoft.VisualStudio.Shell.Interop.8.0" Version="$(MicrosoftVisualStudioShellPackagesVersion)" PrivateAssets="all" />
-   <PackageReference Include="Microsoft.ServiceHub.Framework" Version="$(MicrosoftServiceHubFrameworkVersion)" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.ServiceHub.Framework" Version="$(MicrosoftServiceHubFrameworkVersion)" PrivateAssets="all" />
```

Comment:
The vulnerable Newtonsoft.json is referenced by `Microsoft.VisualStudio.ProjectSystem.Managed` and related packages.
I try to update these packages but it introduces some assembly version mismatch.
![image](https://user-images.githubusercontent.com/24360909/178414091-884af38c-b1d8-465b-828d-d377ddb9ea4d.png)

This looks like a big version update so I choose to reference Newtonsoft.Json directly